src/lustre_dev_tools/error.gleam

```diff
@@ -517,6 +519,19 @@ you were trying to do when you ran into this issue.
   |> string.replace("{path}", path)
 }
 
+fn otp_too_old(version: String) -> String {
+  let message =
+    "
+It looks like you're running an OTP version that is not supported by the dev
+tools: {version}.
+
+You should upgrade to OTP 26 or newer to run this command.
```

Comment:
Could we drop a link to Gleam's install instructions for erlang here: https://gleam.run/getting-started/installing/#installing-erlang

Reply:
Sure!